src/cascadia/Remoting/WindowManager.cpp

```diff
@@ -250,9 +263,13 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
         // Here, we're the king!
         //
         // This is where you should do any additional setup that might need to be
-        // done when we become the king. THis will be called both for the first
+        // done when we become the king. This will be called both for the first
         // window, and when the current monarch dies.
 
+        auto weakThis{ get_weak() };
+
+        _monarch.WindowCreated({ get_weak(), &WindowManager::_WindowCreatedHandlers });
+        _monarch.WindowClosed({ get_weak(), &WindowManager::_WindowClosedHandlers });

```

Comment:
```suggestion
        _monarch.WindowCreated({ weakThis, &WindowManager::_WindowCreatedHandlers });
        _monarch.WindowClosed({ weakThis, &WindowManager::_WindowClosedHandlers });
```

Reply:
Looks like I can't do this one since the `TypedEventHandler` constructor wants to take the weak_ref by move. I'll get rid of the `auto weakThis {...}` above that was probably from me trying to do this before.